pkg/sidecar/sidecar_linux.go

```diff
@@ -19,6 +19,7 @@ const (
 var runners = map[string]func() (Reactor, error){
 	"docker": NewDockerReactor,
 	"k8s":    NewK8sReactor,
+	"mock":   NewMockReactor,
```

Comment:
This will enable us to use the sidecar in CI/kind (although not in any meaningful way)